src/client/public/scripts/pages/github-app-config.ts

```diff
@@ -0,0 +1,202 @@
+import {html} from '../../../../../node_modules/lit-html/lib/lit-extended.js';
+import {TemplateResult} from '../../../../../node_modules/lit-html/lit-html.js';
+import * as api from '../../../../types/api.js';
+import {BaseElement, property} from '../components/base-element.js';
+
+const DEFAULT_CONFIG = `{
+  // Add custom configuration here
+}`;
+
+class GithubAppConfig extends BaseElement {
```

Comment:
I would probably rename this. See below comments on separating the app a little from org config.